packages/ses/README.md

```diff
@@ -1,7 +1,17 @@
-# Secure ECMAScript (SES)
+# SES
 
-Secure ECMAScript (SES) is an execution environment that provides fine-grained
-sandboxing with Compartments.
+SES is a slight modification to the JavaScript language that enables programs
+to defend their own correctness, mostly by disabling dangerous features.
```

Comment:
"Defend their own correctness" is correct, but doesn't appeal very much to me. "Correctness" sounds like it's using formal verification or something arcane and complicated. What about "SES disables dangerous features inherent in JavaScript and makes it possible to run untrusted code securely."